.github/workflows/repository.yml

```diff
@@ -75,17 +76,17 @@ jobs:
         run: bundle exec ruby ./scripts/Algolia.rb
         env:
           ALGOLIA_APP_ID: ${{ secrets.ALGOLIA_APP_ID }}
-          ALGOLIA_INDEX_NAME: ${{ secrets.ALGOLIA_INDEX_NAME }}
+          ALGOLIA_INDEX_NAME : ${{ vars.ALGOLIA_INDEX_NAME }}
```

Comment:
Is there a reason this was a changed from `secrets` to `vars` when everything else remained the same?

Reply:
I didn't really see a reason for encrypting this variable as it's publicly available already.